NEWS_DEV.txt

```diff
@@ -104,6 +105,7 @@ Others:
 
 
 Other more detailed changes:
+ - Use of 64-bit indices in sparse.AddSD where necessary
```

Comment:
This is the title of the PR, but there was more stuff in it. I think this would better describe it.
- Use of 64-bit indexing in sparse ops to allow matrix with more then 2**31-1 elements.